template/src/main/twirl/ch.epfl.scala.index.views/artifact.scala.html

```diff
@@ -1,9 +1,9 @@
-@import ch.epfl.scala.index.model.Project
```

Comment:
This was a fix for IntelliJ - in view files IntelliJ cant resolve the imports by `._` 
Maybe there is a workaround which i don't know ...